metron-interface/metron-rest/src/main/scripts/metron-rest

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# metron rest application service
+# description: rest application
+# processname: metron-rest
+#
+if [ -f /etc/rc.d/init.d/functions ]; then
+    . /etc/rc.d/init.d/functions
+fi
+
+NAME=metron-rest
+DESC="Metron REST Application"
+SCRIPTNAME=/etc/init.d/$NAME
+LOGFILE="/var/log/metron-rest.log"
+METRON_VERSION=0.3.1
+METRON_HOME=/usr/metron/$METRON_VERSION
+METRON_REST_PORT=${METRON_REST_PORT:-8082}
+DAEMON="java -jar $METRON_HOME/lib/metron-rest-$METRON_VERSION.jar --spring.config.location=$METRON_HOME/config/application.yml --spring.profiles.active=dev --server.port=$METRON_REST_PORT"
```

Comment:
The only thing that profile does is select JDBC authentication, set the database to H2 and load a test user.  This could potentially warrant it's own discussion:  how should we authenticate Metron services installed through Ambari MPack?  Should we tackle that here or create a follow-on Jira and possibly start a discussion?